pyuavcan/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -86,6 +86,10 @@ def __init__(
             - Interface ``usb2can`` is described in https://python-can.readthedocs.io/en/stable/interfaces/usb2can.html.
               Example: ``usb2can:ED000100``
 
+            - Interface ``canalystii`` is described in https://python-can.readthedocs.io/en/stable/interfaces/
+canalystii.html.
```

Comment:
Now the doc build is failing because of this:

```suggestion
            - Interface ``canalystii`` is described in
              https://python-can.readthedocs.io/en/stable/interfaces/canalystii.html.
```

Reply:
yeah. black did not process this line, now I know how to process this case. thank you for your patience.